bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -271,8 +271,10 @@ public void run() {
 
   public synchronized BigtableClient getDataClient() throws IOException {
     if (this.client == null) {
-      Channel channel = createChannel(options.getDataHost(), options.getChannelCount());
-      this.client = new BigtableGrpcClient(channel, batchPool, options.getRetryOptions());
+      Channel generalChannel = createChannel(options.getDataHost(), options.getChannelCount());
+      Channel scanChannel = createChannel(options.getDataHost(), 1);
```

Comment:
Yes.  Scans run a long time over a single channel.  We encourage a lot of mutations asynchronously, so mutations benefit from more channels.